ci-operator/step-registry/openshift-extended/test/disasterrecovery/openshift-extended-test-disasterrecovery-ref.yaml

```diff
@@ -28,6 +28,9 @@ ref:
       After test filsters are executed, it is filters to select case for certain moudle.
       the formart: <moudle_name1>:<filters1>#<moudle_name2>:<fitlers2>#<moudle_nameN>:<fitlersN>
       the example: OLM:V1&;~Catalog&#Operator_SDK:~opm&;Cve&
+  - name: TEST_SET_DISASTERRECOVERY
```

Comment:
@wangke19 it is not needed. if TEST_IMPORTANCE take default value `""`, it means it take all importances.

I know you want to get LEVEL0. it should be using filter. see below comments.

Reply:
please remove this env which is useless.

Reply:
@wangke19 ^^

Reply:
Updated

Reply:
@wangke19 I do not get your updated code. the TEST_SET_DISASTERRECOVERY is still in this yaml file.

Reply:
Removed it.